source/globalCommands.py

```diff
@@ -1956,6 +1961,57 @@ def script_reportOrShowFormattingAtCaret(self, gesture):
 		elif repeats == 1:
 			self.script_showFormattingAtCaret(gesture)
 
+	@script(
+		description=_(
+			# Translators: the description for the reportDetailsSummary script.
+			"Report summary of any annotation details at the current location."
+		)
+	)
+	def script_reportDetailsSummary(self, gesture):
+		"""Report the annotation details summary for the single character under the caret or the current nav object.

```

Comment:
This comment is not valid anymore since we're using the focus object rather than nav object.

Reply:
Ah, yep. I missed that!